parts/linux/cloud-init/artifacts/components.json

```diff
@@ -530,12 +530,12 @@
             "versions": [
               "1.27.15",
               "1.27.16",
-              "1.28.12",
               "1.28.13",
-              "1.29.7",
+              "1.28.14",
               "1.29.8",
-              "1.30.3",
-              "1.30.4"
+              "1.29.9",
+              "1.30.4",
+              "1.30.5"
```

Comment:
Hi @HuShaoRu we just had a major update about the components.json because we onboarded Renovate to auto-update the components. We still support 2 patch versions for every minor. Could you please follow the readme or existing examples in this json (after pulled the latest master) to change them to the expected format.? After that Renovate will automatically create PR to update the components. Let me know if you have any question. Thanks.

Reply:
updated. (just change the `latestVersion` and `previousLatestVersion` of ContainerImage `kube-proxy` and Package `kubernetes-binaries`)

Reply:
also updated tests in `cse_helpers_spec.sh`

Reply:
Thanks for making the changes. lgtm.
The latest change I made to master branch was to decouple the unit-test with the actual components.json changes. So moving on we no longer need to change the `cse_helpers_spec.sh` along with the versions update. That will make our life easier.

Reply:
Another thing is that we enabled auto-update for all the container images listed in `components.json`. So moving all, Renovate bot will automatically create PR to update each container image. Can I add you (or people in your team as assignee for kube-proxy?) This is the responsibility of an assignee. https://github.com/Azure/AgentBaker/blob/master/.github/README-RENOVATE.md#what-is-the-responsibility-of-a-pr-assignee

Reply:
yes, you can add me(@HuShaoRu ), @wenhug @gaopenghigh as assignees for kube-proxy